build/dsc/templates/dsc_type_spec.rb.erb

```diff
@@ -54,8 +64,8 @@ describe Puppet::Type.type(:dsc_<%= resource.friendlyname.downcase %>) do
     dsc_<%= resource.friendlyname.downcase %>[:dsc_<%= property.name.downcase %>] = '<%= value.downcase %>'
     expect(dsc_<%= resource.friendlyname.downcase %>[:dsc_<%= property.name.downcase %>]).to eq('<%= value.downcase %>')
   end
+  <%end%>
 <%        if property.name.downcase == 'ensure' -%>
-
```

Comment:
FYI we've got some whitespace noise being injected into generated specs due to the removal of this line.